DMProtocol/ProtocolPackageCreator.cs

```diff
@@ -36,6 +36,22 @@ public static class Factory
             /// -or-
             /// The protocol does not have a version specified in the Version tag.</exception>
             public static async Task<IAppPackageProtocol> FromRepositoryAsync(ILogCollector logCollector, string repositoryPath)
+            {
+                return await FromRepositoryAsync(logCollector, repositoryPath, String.Empty);
+            }
+            /// <summary>
+            /// Creates an <see cref="IAppPackageProtocol"/> instance from the specified repository with the specified name and version.
+            /// </summary>
+            /// <param name="logCollector">The log collector.</param>
+            /// <param name="repositoryPath">The path of the repository that contains the Protocol solution.</param>
+            /// <returns>The <see cref="IAppPackageProtocol"/> instance.</returns>
+            /// <exception cref="ArgumentNullException"><paramref name="logCollector"/>, <paramref name="repositoryPath"/> is <see langword="null"/>.</exception>
+            /// <exception cref="DirectoryNotFoundException">The directory specified in <paramref name="repositoryPath"/> does not exist.</exception>
+            /// <exception cref="AssemblerException">Project with name could not be found.</exception>
+            /// <exception cref="InvalidOperationException">The protocol does not have a name specified in the Name tag.
+            /// -or-
+            /// The protocol does not have a version specified in the Version tag.</exception>
+            public static async Task<IAppPackageProtocol> FromRepositoryAsync(ILogCollector logCollector, string repositoryPath, string overrideVersion)
```

Comment:
argument overrideVersion not documented. Should it not be overriddenVersion or versionOverride? as overrideVersion reads as if it's a boolean.